packages/gatsby/cache-dir/api-ssr-docs.js

```diff
@@ -99,17 +99,17 @@ exports.onRenderBody = true
  * replace head components to be rendered in your `html.js`. This is useful if
  * you need to reorder scripts or styles added by other plugins.
  * @param {Object} $0
- * @param {Array} $0.getHeadComponents Returns the current `headComponents` array.
+ * @param {Array<ReactNode>} $0.getHeadComponents Returns the current `headComponents` array.
```

Comment:
does ReactNode work? It's not defined anywhere but maybe it's just to give a better understanding to people read it.

on line https://github.com/gatsbyjs/gatsby/pull/12087/files#diff-2ad21b5b5ed90fd9c676b17f828cbea5R16 we were using `@returns {string[]}` there is no difference in parsing but just not sure what syntax we like

Reply:
Those are really just to create API reference at this point. Only real thing that this does is actually display `ReactNode` in https://www-doc-preview--tender-curie-facda8.netlify.com/docs/ssr-apis/#onPreRenderHTML

If this would be used for providing typings than yeah, we probably would need to look into actually importing React types somewhere to make it actually work.

Reply:
As for `Array<ReactNode>` or `ReactNode[]` - I don't really have preference and can switch no problem

Reply:
I like the Array syntax but who am i 😜